app/uk/gov/hmrc/eoricommoncomponent/frontend/views/subscription/reg06_eori_already_linked.scala.html

```diff
@@ -17,20 +17,20 @@
 @import org.joda.time.DateTime
 @import uk.gov.hmrc.eoricommoncomponent.frontend.forms.FormUtils.dateTimeFormat
 @import uk.gov.hmrc.eoricommoncomponent.frontend.views.html._
+@import uk.gov.hmrc.eoricommoncomponent.frontend.views.ServiceName._
 
 
 @this(layout_di: layout)
 @(name: String, processedDate: DateTime)(implicit messages: Messages, request: Request[_])
-@layout_di(messages("cds.reg06.outcome.eori-already-linked.title-and-heading")) {
+@layout_di(messages("cds.reg06.outcome.eori-already-linked.title-and-heading", shortName, name)) {
     <div class="column-two-thirds">
         <div class="govuk-box-highlight">
-            <h1 id="page-heading" class="heading-xlarge" >@messages("cds.reg06.outcome.eori-already-linked.title-and-heading")</h1>
-            <h2 id="page-heading2" class="heading-xlarge" >@messages("cds.reg06.outcome.eori-already-linked.heading", name)</h2>
-            <h3 class="heading-medium" id="processed-date">@messages("cds.reg06.outcome.eori-already-linked.received", dateTimeFormat.print(processedDate))</h3>
+            <h1 id="page-heading" class="heading-xlarge" >@messages("cds.reg06.outcome.eori-already-linked.title-and-heading", shortName, name)</h1>
+            <div class="heading-medium" id="processed-date">@messages("cds.reg06.outcome.eori-already-linked.received", dateTimeFormat.print(processedDate))</div>
         </div>
 
-        <h2 class="heading-small" id="use-cds-heading">@messages("cds.subscription.outcomes.eori-already-linked.use-cds-heading")</h2>
-        <p id="use-cds-para">@messages("cds.subscription.outcomes.eori-already-linked.use-cds-para", name)</p>
+        <h2 class="heading-small" id="use-cds-heading">@messages("cds.subscription.outcomes.eori-already-linked.use-cds-heading", shortName)</h2>
+        <p id="use-cds-para">@messages("cds.subscription.outcomes.eori-already-linked.use-cds-para", name, shortName)</p>
```

Comment:
Similar comment here, but I'm not sure if I understand name and shortName correctly

Reply:
I'll change the "name" variable.  In this context it is the name of the organisation/individual, not the service